Multisig/Logic/Ethereum/BlockchainDomainManager.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  BlockchainDomainManager.swift
+//  Multisig
+//
+//  Created by Johnny Good on 1/20/21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+import UnstoppableDomainsResolution
+
+class BlockchainDomainManager {
+    
+    let ens: ENS;
+    let resolution: Resolution?;
+    
+    init() {
+        ens = ENS(registryAddress: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e");
+        self.resolution = try? Resolution(
+            configs: Configurations(
+                cns: NamingServiceConfig(
+                    providerUrl: App.configuration.services.ethereumServiceURL.absoluteString,
+                    network: App.configuration.app.network.rawValue.lowercased()
+                )
+            )
+        );
+    }
+    
+    func resolveUD(_ domain: String) throws -> Address {
+        guard resolution != nil else {
```

Comment:
guard let resolution = resolution else { ... }
No need to use force unwrapping below.